frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -5,269 +5,270 @@
-          </a>&nbsp;&nbsp;&nbsp;&nbsp;
-          <span class="fs-16 fw-light">
-            <strong class="text-light-black">Is Public </strong>
-          </span>
-          <a class="pointer fs-20" (click)="togglePhaseVisibility()">
-            <i class="{{ phaseVisibility.icon }}" aria-hidden="true"> </i>
-          </a>&nbsp;&nbsp;&nbsp;&nbsp;
-          <span class="fs-16 fw-light">
-            <strong class="text-light-black">Submission Visibility </strong>
-          </span>
-          <a class="pointer fs-20" (click)="toggleSubmissionVisibility()">
-            <i class="{{ submissionVisibility.icon }}" aria-hidden="true"> </i>
-          </a>
+      <!-- Edit Leaderboard details -->
+      <mat-tab label="Edit leaderboard details">
+        <div class="row settings-section">
+          <div class="ev-action">
+            <div class="col-sm-6 col-xs-12 col-lg-6 col-lr-pad phase-select-box" *ngIf="isChallengeHost">
+              <div class="row margin-bottom-cancel phase-card">
+                <app-selectphase
+                  [phases]="filteredPhaseSplits"
+                  [phaseSelectionType]="phaseLeaderboardSelectionType"
+                  [phaseSelectionListType]="phaseLeaderboardSelectionListType"
+                  [phaseSplitSelected]="phaseSplitSelected()"
+                  #phasesplitselect
+                >
+                </app-selectphase>
+              </div>
+              <div class="row margin-bottom-cancel phase-card" *ngIf="selectedPhaseSplit">
+                <div class="col s12">
+                  <span class="fs-16 fw-light">
+                    <strong class="text-light-black">Leaderboard Visibility </strong>
+                  </span>
+                  <a class="pointer fs-20" (click)="toggleLeaderboardVisibility()">
+                    <i class="{{ leaderboardVisibility.icon }}" aria-hidden="true"> </i>
+                  </a>&nbsp;&nbsp;&nbsp;&nbsp;
+                </div>
+              </div>
+            </div>
+          </div>
         </div>
-       </div>
-    </div>
-  </div>
+      </mat-tab>
 
-    <!-- Leaderboard Edit Section-->
-    <div class="row settings-section">
-      <div class="col-lg-3 col-md-4 col-sm-3 col-xs-3">
-        <span class="fw-light fs-18">Edit leaderboard details</span>
-      </div>
-      <br />
-      <div class="ev-action">
-        <div class="col-sm-6 col-xs-12 col-lg-6 col-lr-pad phase-select-box" *ngIf="isChallengeHost">
-          <div class="row margin-bottom-cancel phase-card">
-          <app-selectphase
-            [phases]="filteredPhaseSplits"
-            [phaseSelectionType]="phaseLeaderboardSelectionType"
-            [phaseSelectionListType]="phaseLeaderboardSelectionListType"
-            [phaseSplitSelected]="phaseSplitSelected()"
-            #phasesplitselect
-          >
-          </app-selectphase>
+      <!-- Edit Evaluation Script and Criteria & Worker logs -->
+      <mat-tab label="Evaluation Script and Guidelines"> 
```

Comment:
Let's keep it as `Evaluation script and guidelines` in order to be consistent in tab names

Reply:
Sure!